docs/INSTALL.K8S.md

```diff
@@ -26,12 +26,20 @@ nohup ./kube-controller-manager --master=127.0.0.1:8080 \
 nohup ./kube-scheduler --master=127.0.0.1:8080 --v=2 2>&1 > /dev/null &
 ```
 
-On the minions, start the kubelet specifying that the network plugin is of type
-CNI. e.g:
+On the minions, you need to download a few upstream CNI plugins
+
+```
+mkdir -p /opt/cni/bin && cd /opt/cni/bin
+wget https://github.com/containernetworking/cni/releases/download/v0.2.0/cni-v0.2.0.tgz
+tar xfz cni-v0.2.0.tgz
+```
+
+On minions, start the kubelet specifying that the network plugin is of type
+CNI and the network plugin directory to be /etc/cni/net.d. e.g:
 
 ```
 nohup ./kubelet --api-servers=http://10.33.74.22:8080 --v=2 --address=0.0.0.0 \
---enable-server=true --network-plugin=cni 2>&1 > /dev/null &
+--enable-server=true --network-plugin=cni --network-plugin-dir=/etc/cni/net.d  2>&1 > /dev/null &
```

Comment:
Without the --network-plugin-dir, I did not see it working.  There are issues with CNI integration and k8s with a few backwards incompatible changes so far.